cmd/algons/dnsCmd.go

```diff
@@ -181,6 +181,19 @@ var exportCmd = &cobra.Command{
 	},
 }
 
+func doAddTXT(from string, to string) error {
+	cfZoneID, cfToken, err := getClouldflareCredentials()
+	if err != nil {
+		return fmt.Errorf("error getting DNS credentials: %v", err)
+	}
+
+	cloudflareDNS := cloudflare.NewDNS(cfZoneID, cfToken)
+
+	const priority = 1
+	const proxied = false
+	return cloudflareDNS.CreateDNSRecord(context.Background(), "TXT", from, to, cloudflare.AutomaticTTL, priority, proxied)
```

Comment:
`doAddDNS` uses `SetDNSRecord`. Should that be used here as well? I wonder if that allows you to create or update. What happens with this command if you update something that already exists?

Reply:
Is this because we don't support adding a single record, and the entire tree needs to be rebuilt if we do?

Reply:
You're correct that I don't intend to use this tool as a way to add/del certain entries or maintain records--simply put up and tear down.

As for why I needed to use Create instead of Set, [SetDNSRecord](https://github.com/algorand/go-algorand/blob/master/tools/network/cloudflare/cloudflare.go#L75) calls `UpdateDNSRecord` if entries exist. This meant that on a single TXT entry would ever be created when using this and it would just be updated and overwritten. Using Create just defaults to creating new entries every time which is what I intended.